flight_phase_files/HALO_RF04_20200126_info.yaml

```diff
@@ -21,71 +26,366 @@ remarks:
 - Four coordinated straight legs over Meteor.
 - Cirrus clouds in 14-17 km present for almost full flight.
 segments:
-- kind: straight_leg
+- kinds:
+  - straight_leg
   name: super_curtain
+  irregularities: []
+  segment_id: HALO-0126sc1
```

Comment:
Segment ID should combine flight id with underscore: HALO-0126sc1 -> HALO-0126_sc1